docs/usage_instructions.md

```diff
@@ -13,31 +13,140 @@ Where:
 
 [action] is the action you wish to perform. This may vary from object type to object type (see below), but is generally one of: ["get", "create", "modify", "list"]
 
-[params]
+[params] represents command specific parameters. See below for more detail.  
+--id to specify id  
+--name or -N to specify the name  
+--active or -A to specify set activity
 
 ## Interfacing with projects
 The following commands can be used to create, list, and get projects.
 
 
 #### Create
 Input:  
+
+```
 ringling-cli localhost project create -N name_here
+```
 
-Output:  
+Output:
+
+```
 Project name_here created successfully  
 Project ID: 4
+```
 
 #### List
-Input:  
+Input:
+
+```
 ringling-cli localhost project list
+```
 
-Output:  
+Output:
+
+```  
 {'projects': [{'project_id': 1, 'project_name': 'test'},  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;{'project_id': 2, 'project_name': 'test'}]}  
+              {'project_id': 2, 'project_name': 'test2'}]}  
+```
 
 #### Get
 Input:  
+
+```
 ringling-cli localhost project get --id 1
+```
 
 Output:  
+
+```
 {'project_id': 1, 'project_name': 'test'}
+```
+
+## Interfacing with Parameter Sets
+The following commands can be used to create, list, modify, and get paramater sets.
+
+#### Create
```

Comment:
I think we should only support the case where a path to a JSON file is provided.  Too complicated to support all of the options -- but this was a great effort!